pkgs/dartpad_ui/web/ga.js

```diff
@@ -0,0 +1,11 @@
+(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+ga('create', 'UA-26406144-18', {
+    cookieFlags: 'secure;samesite=none',
+    cookieDomain: 'auto'
+});
+ga('set', 'referrer', document.referrer.split('?')[0]);
+ga('send', 'pageview');
```

Comment:
missing newline?

Reply:
Sorry, I'm currently traveling and want to land this asap. Feel free to send a pull request if you're able to.